### PR TITLE
Updated instructions for installing and running EVW Self-Serve locally…

### PR DESCRIPTION
**Pull request by paulwaitehomeoffice**
… with mocks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,17 +11,19 @@ A tiny, HOF-based form to allow Electronic visa waiver users to update their tra
...
+$ mongod &
 $ npm install
-$ npm run dev
+$ npm run dev:mock
 ```
 
-Go to http://localhost:8080/update-journey-details
+Go to http://localhost:8080/find-your-application
+
+When you’re asked to check your e-mail, go to http://localhost:8080/update-journey-details/how-will-you-arrive?evwNumber=VALID&token=VALID
```
Comment (sulthan-ahmed) [inline README.md]: I don't think you need the bit about `When you're asked to check your email` You can just go straight to that url

File: README.md [hunk 2]
```diff
@@ -11,17 +11,19 @@ A tiny, HOF-based form to allow Electronic visa waiver users to update their tra
...
 - [NodeJS](https://nodejs.org/en/)
 - npm (bundled with node)
-- [Redis server](http://redis.io/topics/quickstart) running on the default port
+- [MongoDB](https://www.mongodb.com) running on the default port
 
 ### Installing
 
 ```bash
-$ redis-server &
+$ mongod &
```
[the remark this answers is not in the document]
Comment (paulwaitehomeoffice) [reply README.md]: Hey yeah good shout. No the `&` is just if you want to run mongod in the background. Personally I prefer not to.

File: README.md (hunk 1 above)
Comment (sulthan-ahmed) [reply README.md]: Is it linked with the token? I know it is in the real app but in this situation?

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sulthan-ahmed) [reply README.md]: ahh, I've been trying to work that out now I know thanks

File: README.md (hunk 1 above)
Comment (paulwaitehomeoffice) [reply README.md]: Hm — actually I don’t think `token` is required, at least just to click through here. It works for me with just `evwNumber`.

Comment (sulthan-ahmed) [reply README.md]: I think it's fine keep that link as it is

File: .npmrc [hunk 3]
```diff
@@ -0,0 +1,4 @@
+registry = http://artifactory.registered-traveller.homeoffice.gov.uk/artifactory/api/npm/npm/
+_auth = Ym9ibWVyZWRpdGg6QVA5V3Q0RHh0U29OamhLN3JvV2hwVTloVk1G
+always-auth = true
+email = paul.waite@digital.homeoffice.gov.uk
```
Comment (sulthan-ahmed) [inline .npmrc]: are you sure we all need your npmrc for this to work?

**Pull request merged**